starters/shopify-meilisearch/views/homepage/categories-section.tsx

```diff
@@ -1,62 +1,31 @@
-import { meilisearch } from "clients/search"
-import { Skeleton } from "components/ui/skeleton"
-import { env } from "env.mjs"
-import { unstable_cache } from "next/cache"
-import Image from "next/image"
-import Link from "next/link"
-import { getDemoCategories, isDemoMode } from "utils/demo-utils"
+import { CategoryCard } from "components/category-card"
+import { PlatformCollection } from "lib/shopify/types"
+import { cn } from "utils/cn"
 
-export async function CategoriesSection() {
-  const categories = await getCategories()
+export async function CategoriesSection({ categories }: { categories: Pick<PlatformCollection, "handle" | "title" | "id" | "image">[] }) {
```

Comment:
This component could just fetch categories on its own

Reply:
done